files/mon_grants.sql

```diff
@@ -0,0 +1,15 @@
+GRANT USAGE ON SCHEMA MonMetrics TO mon_persister;
```

Comment:
I would prefer to see this combined with the users, that way users and grants are in one file.